test/unit/model/define.test.js

```diff
@@ -49,9 +49,15 @@ describe(Support.getTestDialectTeaser('Model'), () => {
     it('should throw when the attribute name is ambiguous with $nested.attribute$ syntax', () => {
       expect(() => {
         current.define('foo', {
-          $id$: DataTypes.INTEGER,
```

Comment:
Maybe we still want to test $id$?

Reply:
The code would have to be a bit convulter to have $id and id$ throw but not $id$ but I can add it if you prefer :)

Reply:
No, it's fine like this. Let's just merge it